File: C++/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P07_WorldSwimmingRecord.cpp
#include <iostream>
#include <stdio.h>  
#include <math.h>       
using namespace std;

int main() {

	double worldRecord, distance, time;
	cin >> worldRecord >> distance >> time;

	double slowDown = floor(distance / 15) * 12.5;

	double totalTime = (distance * time) + slowDown;

	if (totalTime < worldRecord) {
		printf("Yes, he succeeded! The new world record is %.2lf seconds.\n", totalTime);
	} else {
		printf("No, he failed!He was %.2lf seconds slower.", totalTime - worldRecord);
	}

	return 0;
}

File: Python/M01_ProgrammingBasics/L02_ConditionalStatements/Exercises/Solutions/P07_WorldSwimmingRecord.py
import math

record = float(input())
distance = float(input())
speed = float(input())

normal = speed * distance
slow_down_time = distance / 15
slow_rounded = math.floor(slow_down_time)
slow = slow_rounded * 12.5
real_time = normal + slow
needed = real_time - record

if real_time < record:
    print(f'Yes, he succeeded! The new world record is {real_time:.2f} seconds.')
else:
    print(f'No, he failed! He was {needed:.2f} seconds slower.')